src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -66,7 +66,7 @@ def get_resources_in_resource_group(resource_group_name, resource_type=None):
     from azure.cli.core.commands.client_factory import get_mgmt_service_client
     rcf = get_mgmt_service_client(ResourceType.MGMT_RESOURCE_RESOURCES)
     filter_str = "resourceType eq '{}'".format(resource_type) if resource_type else None
-    return list(rcf.resource_groups.list_resources(resource_group_name, filter=filter_str))
+    return list(rcf.resources.list_by_resource_group(resource_group_name, filter=filter_str))
```

Comment:
@derekbekoe the rename wasn't done for older API versions. How do you propose we handle this?